GOVERNANCE.md

```diff
@@ -8,7 +8,19 @@ This document describes how the Tessel Project is organized.
 
 The Tessel project follows the mission and philosophies outlined [here](/MISSION.md).
 
-The Tessel project is organized in a tiered system. Members of the Steering Committee jointly govern and give high-level guidance to the project. Team Members, chosen by the Steering Committee, manage the day-to-day of community management and technical review. Anyone is welcome to be a Contributor and add to the discussion or to technical assets. These roles and the Project’s processes are outlined in more detail below.
+The Tessel project is organized in a tiered system, as illustrated in this Venn diagram:
```

Comment:
I would just call it a diagram since it doesn't seem to fit the description of a traditional Venn diagram (although I see what you mean):

![](https://upload.wikimedia.org/wikipedia/commons/thumb/3/39/Венов_дијаграм.svg/440px-Венов_дијаграм.svg.png)

This is really being nit picky.